modules/mobile/spec/models/user_accessible_services_spec.rb

```diff
@@ -383,41 +383,16 @@
       context 'when user does not have vet360 access' do
         let(:user) { build(:user, :loa3, vet360_id: nil) }
 
-        it 'does not include userProfileUpdate' do
-          expect(user_services.authorized).not_to include(:userProfileUpdate)
+        it 'is false' do
+          expect(user_services.service_auth_map[:userProfileUpdate]).to be(false)
         end
       end
 
       context 'when user does have vet360 access' do
-        it 'includes userProfileUpdate' do
-          expect(user_services.authorized).to include(:userProfileUpdate)
+        it 'is true' do
+          expect(user_services.service_auth_map[:userProfileUpdate]).to be_truthy
         end
       end
     end
   end
-
```

Comment:
I'm open to this, but FWIW, I wasn't asking you to remove this. I was suggesting you add a similar simple test for the authorized method. They're both public methods so I figured they should at least have a very perfunctory test. But I can also see the argument that they're effectively tested through the request specs so 🤷.